### PR TITLE
Update Ubuntu Summit logo

### PR DESCRIPTION
**Pull request by petesfrench**
https://warthogs.atlassian.net/browse/WD-5291

### CODE REVIEW

**Comment by petesfrench** [comment]
This work can close this other PR https://github.com/canonical/canonical-indico-themes/pull/37

**Comment by carkod** [comment]
@kewisch can you take a look? Maybe this one is better than my PR?

**Review by carkod — APPROVED** [review]
LGTM

File: indico_themes_canonical/static/js/ubuntu_summit.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ function buildHeaderStrip() {
   headerLogoCont.classList.add("headerLogoCont");
   const headerLogo = document.createElement("img");
   headerLogo.classList.add("headerLogo");
-  headerLogo.src = "https://assets.ubuntu.com/v1/88af8658-ubuntu+summit-illustration.svg"
+  headerLogo.src = "https://assets.ubuntu.com/v1/5a965c5c-summit-2023-logo.png"
```
Comment (kewisch) [inline indico_themes_canonical/static/js/ubuntu_summit.js:118]: This change will still change the Ubuntu Summit 2022 logo to the 2023 one. Was it intended to merge this, or was the main purpose fixing the error message colors?

**Comment by petesfrench** [comment]
@kewisch The error boxes was a flyby change

**Comment by kewisch** [comment]
Sounds good, thanks! Still though I don't think this fixes the issue correctly. Given the 2023 Summit is over we can probably get away without updating the logo on the theme, we used the header logo setting in the Indico UI for this year and it worked ok.

As we get closer to the 2024 event we should reserve some time to ensure logo and lines placement is up to par with expectations.

Can you revert the asset image changes and positioning so we don't accidentally change the 2022 logo?

**Comment by petesfrench** [comment]
@kewisch Yes, it is fine to use the UI to update the logos. But this should be communicated with the design team responsible for making the logo as this will effect alignment. This issue was raise due to a miss-aligned logo.

I will make another PR reverting the change.

**Pull request merged**